packages/api/tsserver/tsservers.mts

```diff
@@ -0,0 +1,68 @@
+import { spawn } from 'child_process';
+import { TsServer } from './tsserver.mjs';
+
+/**
+ * This object is responsible for managing multiple tsserver instances.
+ */
+export class TsServers {
+  private servers: Record<string, TsServer> = {};
+
+  get(id: string) {
+    const server = this.servers[id];
+
+    if (!server) {
+      throw new Error(`tsserver for ${id} does not exist.`);
```

Comment:
I feel like throwing is a bit heavy here, how about returning `null`? I guess we don't really have a policy for this kind of thing, and there's a `has()` method that can be used to check